app/CPReflectionService/RBObject+CocoaPods.rb

```diff
@@ -88,7 +88,7 @@ def self.pod_sources
       sources_manager.all.map { |source|
         { source.name => source.url }
 
-      }.reduce &:merge
+      }.reduce(&:merge).select { |key, value| key.is_a?(String) && value.is_a?(String) }
```

Comment:
I'm guessing all you have to do here is change this to `reduce({}, &:merge)` -- if there are no sources, it'll return `nil` as-is